### ISSUE TITLE
Normalized coverage 

### ISSUE THREAD

**Issue by npatel22526** [brentp/mosdepth#67]
Hello, 

This is great tool for coverage calculation and quite useful for CNV validation. 
For part of QC analysis I usually perform sex check using chrX and chrY coverage , which gets tricky sometimes when all samples are not sequenced at same depth. So to overcome the variable depth  issue I tend to normalize chrX and chrY coverage with mean autosomal coverage of individual samples , is it something easily done by the output file produced by mosdepth ?
Basically I need the same plot generated by "plot-dist.py" but X-axis being normalized coverage instead of raw coverage.

Thanks, 
Nick

**Comment by brentp** [comment]
this would be easy to add inside of plot-dist.py. I would recommend to do it there. Or you can normalize the median X coverage by the median total coverage. This is outside of the scope of mosdepth itself.
You could use [indexcov](https://github.comb/brentp/goleft) to get a quick view of copy-number on X and Y.
